library/core/src/main/java/com/google/android/exoplayer2/drm/FrameworkMediaDrm.java

```diff
@@ -153,8 +153,9 @@ public KeyRequest getKeyRequest(
     }
 
     // Prior to API level 26 the ClearKey CDM only accepted "cenc" as the scheme for MP4.
+    // For APIs less than 27 ClearKey UUID is changed to Common PSSH UUID in the constructor
     if (Util.SDK_INT < 26
-        && C.CLEARKEY_UUID.equals(uuid)
+        && C.COMMON_PSSH_UUID.equals(uuid)
```

Comment:
I don't think this fix is sufficient in all cases, because there are two other `C.CLEARKEY_UUID.equals(uuid)` checks in this class. I'm going to submit a cleaner fix that makes it so `uuid` is set to `C.CLEARKEY_UUID` in the first place.